### DISCUSSION TITLE
Nova assets not loaded on Laravel running in subfolder

### DISCUSSION THREAD

**Discussion by kichetof** [laravel/nova-issues#3991] [Q&A] [answered]
- Laravel Version: 9.8.1
- Nova Version: 4.2.1
- PHP Version: 8.1.4
- Database Driver & Version: Mysql
- Operating System and Version: Docker Laravel Sail
- Browser type and version: Brave 1.37.109

### Description:
Unable to load Nova assets when Laravel run in subfolder.
APP_URL seems to not be used *(as Laravel without custom boot)*.

Assets should be loaded as APP_URL, ex. `localhost/backend/vendor/nova/vendor.js` but currently is `localhost/vendor/nova/vendor.js` without subfolder

### Detailed steps to reproduce the issue on a fresh Nova installation:

_.env_
```php

APP_URL=localhost/backend
```

_app/Providers/AppServiceProvider.php_
```php
/**
* Bootstrap any application services.
*
* @return void
*/
public function boot()
{
   url()->forceRootUrl(config('app.url'));
}
```

**Comment by kichetof** [comment]
Hi guys,
Do you have any update how I can solve nova inside a subfolder as **example.com/backend/nova** instead of _example.com/nova_ ?

_My url is already a subdomain and my goal is to keep this name to not disturb users (team_a.example.com/backend/nova) and I can't have a more depth subdomain as backend.team_a.example.com_

**Comment by theshubhamchugh** [comment]
Any solution for this?

**Comment by davidhemphill** [comment, marked as answer]
Please see: https://github.com/laravel/nova/pull/296#issuecomment-474514093
